pkg/sfu/sequencer_test.go

```diff
@@ -97,3 +91,83 @@ func Test_sequencer_getNACKSeqNo(t *testing.T) {
 		})
 	}
 }
+
+func Test_packetMeta_VP8(t *testing.T) {
+	p := &packetMeta{}
+
+	vp8 := &buffer.VP8{
+		FirstByte:        25,
+		PictureIDPresent: 1,
+		PictureID:        55467,
+		MBit:             true,
+		TL0PICIDXPresent: 1,
+		TL0PICIDX:        233,
+		TIDPresent:       1,
+		TID:              13,
+		Y:                1,
+		KEYIDXPresent:    1,
+		KEYIDX:           23,
+		HeaderSize:       6,
+		IsKeyFrame:       true,
+	}
+
+	p.packVP8(vp8)
+
+	// booleans are not packed, so they will be `false` in unpacked.
+	// Also TID is only two bits, so it should be modulo 3.
+	expectedVP8 := &buffer.VP8{
+		FirstByte:        25,
+		PictureIDPresent: 1,
+		PictureID:        55467,
+		MBit:             false,
+		TL0PICIDXPresent: 1,
+		TL0PICIDX:        233,
+		TIDPresent:       1,
+		TID:              13 % 3,
+		Y:                1,
+		KEYIDXPresent:    1,
+		KEYIDX:           23,
+		HeaderSize:       6,
+		IsKeyFrame:       false,
+	}
+	unpackedVP8 := p.unpackVP8()
+	require.True(t, reflect.DeepEqual(expectedVP8, unpackedVP8))
+
+	// short picture id and no TL0PICIDX
```

Comment:
Looks like you have 2 tests in 1.

Reply:
Yeah, I just keep all related checks in the same test.

Reply:
IMHO better to have 2 different tests. In that way tests won't interfere to each other. And won't hide each other results. But as you prefer.